src/robusta/core/discovery/discovery.py

```diff
@@ -205,6 +205,14 @@ def is_pod_ready(pod: V1Pod) -> bool:
     return False
 
 
+def is_pod_finished(pod: V1Pod) -> bool:
+    try:
+        # all containers in the pod have terminated, this pod should be removed by GC
+        return pod.status.phase.lower() in ['succeeded', 'failed']
```

Comment:
Are we deliberately including "Unknown" and "Pending" phases?

Reply:
Yes, (unknown , pending or running pods with no owner references)
It wont be an orphan pod if its Unknown or pending. (Since the GC should remove them)

Also since it is a deliberate pod being run by a user he should be receiving alerts due to the pod pending or Unknown phase so it will be important for us to display that for him on the apps screen and not hide it.

Reply:
sounds good, thanks for clarifying